priv/repo/migrations/20220404173205_initial.exs

```diff
@@ -25,6 +27,9 @@ defmodule College.Repo.Migrations.Initial do
       add :first_name, :string, null: false
       add :last_name, :string
       add :email, :string, null: false
+      add :gender, :string
+      add :address, :string
+      add :date_of_birth, :date
```

Comment:
Do you think it'll work properly to add the new fields to the DB?

Reply:
yes ?

i think i need it so i can retrieve the gender , address and date of birth when i get the specified record from the DB

Reply:
would you please send me a screenshot of the students table structure?

Reply:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/81506540/163977408-d226be25-8105-4995-a455-e60330fe14d4.png">

Reply:
hhhh i missread structure to struct 
<img width="800" alt="image" src="https://user-images.githubusercontent.com/81506540/163978368-92724252-1155-4648-8ef4-ebab8751ee0c.png">

Reply:
Thanks!
Now, how did you add the address, date of birth and gender to students and teachers tables?

Reply:
how did you reset ecto?
does it make sense to do this every time we need to add a new field?
we usually create a new migration file with `alter table`
in that way we'll keep our migration files clean and readable and no need to clear the schema migrations table or to change anything on the stable migrations files.

Reply:
oh yes 
it makes more sense 
Thank you